sp/src/game/server/hl2/npc_antlion.cpp

```diff
@@ -3205,9 +3206,13 @@ bool CNPC_Antlion::HandleInteraction( int interactionType, void *data, CBaseComb
 		ApplyAbsVelocityImpulse( vecDir );
 		Flip();
 
-		// If the Combine soldier is a player, don't deal damage on top of flipping
+		// At the beginning of Ashes the non-damaging melee would be a disappointment like in Doom Eternal, so receive concussion damage without bleeding - HEVcrab
```

Comment:
Can you update this comment to simply describe the behavior? Similar to the previous comment.

Reply:
> Can you update this comment to simply describe the behavior? Similar to the previous comment.

Done in [8394d49a](https://github.com/entropy-zero/source-sdk-2013/pull/156/commits/8394d49ac82a294b92d58afc235fbd58a1950e02)